config/diff_test.go

```diff
@@ -888,3 +888,129 @@ func TestDiffRevision(t *testing.T) {
 		test.That(t, diff.UnmodifiedResources, test.ShouldResemble, tc.expectedDiff.UnmodifiedResources)
 	}
 }
+
+func TestDiffJobCfg(t *testing.T) {
+	job1 := config.JobConfigData{
+		Name:     "my-job",
+		Schedule: "5s",
+		Resource: "my-resource",
+		Method:   "my-method",
+	}
+	job2 := config.JobConfigData{
+		Name:     "my-job",
+		Schedule: "* * * * *",
+		Resource: "my-resource",
+		Method:   "my-method",
+		Command: map[string]any{
+			"argument1": float64(12),
+			"argument2": false,
+		},
+	}
+	job3 := config.JobConfigData{
+		Name:     "my-job",
+		Schedule: "3h",
+		Resource: "my-resource",
+		Method:   "my-method",
+		Command: map[string]any{
+			"argument1": float64(12),
+			"argument2": "string",
+		},
+	}
+	job4 := config.JobConfigData{
+		Name:     "my-job-2",
+		Schedule: "3h",
+		Resource: "my-resource",
+		Method:   "my-method",
+		Command: map[string]any{
+			"argument1": float64(12),
+			"argument2": "string",
+		},
+	}
+	job5 := config.JobConfigData{
+		Name:     "my-job-2",
+		Schedule: "3h",
+		Resource: "my-resource",
+		Method:   "my-method",
+	}
+	job6 := config.JobConfigData{
+		Name:     "my-job-2",
+		Schedule: "0 */3 * * *",
+		Resource: "my-resource",
+		Method:   "my-method",
+	}
+
+	jobs1 := []config.JobConfig{
+		{job1},
+		{job2},
+		{job3},
+	}
+	jobs2 := []config.JobConfig{
+		{job3},
+	}
+	jobs3 := []config.JobConfig{
+		{job1},
+		{job2},
+		{job3},
+		{job4},
+	}
+	jobs4 := []config.JobConfig{
+		{job4},
+	}
+	jobs5 := []config.JobConfig{
+		{job5},
+	}
+	jobs6 := []config.JobConfig{
+		{job6},
+	}
+
+	for _, tc := range []struct {
+		Name          string
+		LeftCfg       config.Config
+		RightCfg      config.Config
+		JobsDifferent bool
```

Comment:
Is this supposed to be `JobsEqual`? The first test example has this as `true` for comparing `jobs1` to itself.